src/components/common/color-legend.js

```diff
@@ -91,7 +91,9 @@ const getQuantLegends = (scale, labelFormat) => {
 
   const labels = scale.range().map(d => {
     const invert = scale.invertExtent(d);
-    return `${labelFormat(invert[0])} to ${labelFormat(invert[1])}`;
+    return `${labelFormat(invert[0] ? invert[0] : 0)} to ${
+      invert[1] ? labelFormat(invert[1]) : Infinity
+    }`;
```

Comment:
can we write this in a more readable way? something like this:

```suggestion
    let from = 0;
    let to = Infinity;
    if (labelFormat(invert[0]) {
      from = invert[0];
    }
    if (invert[1]) {
      to = labelFormat(invert[1])
    }
    return `${from} to ${to}`;
```

Reply:
I quickly tested with a point layer from the examples and it worked, but I didn't pass a range there.